generate-protos.sh

```diff
@@ -72,4 +76,6 @@ $PROTOC \
   -I tmp/google-cloudevents/proto \
   $(find tmp/google-cloudevents/proto -name data.proto)
 
-echo "Done"
+echo "- Removing tmp/"
+rm -rf tmp
+echo "~ DONE ✓"
```

Comment:
I'd prefer to stick to ASCII in tool output unless there's a compelling reason not too. There are too many places where this could fail or display weirdly. (I spent quite a lot of time on Friday working out why Travis logging was behaving oddly with .NET Core... let's not introduce another thing that could easily go wrong, with no benefit.)

Reply:
Unicode is pretty common and standard.
Something like CHECK MARK (✓) is common in Node tools for example and should not be an issue here.